kubernetes/helm/collabora-online/values.yaml

```diff
@@ -18,17 +17,18 @@ global:
       port: 9980
       path: "/lool/getMetrics"
     service:
-      name: libreoffice-online
-      type: ClusterIP
+      name: collabora-online
+      type: NodePort # Use ClusterIP instead of NodePort if you are using standalone cluster structure 
       externalPort: 9980
       internalPort: 9980
   ingress: {}
 
-replicaCount: 3
+replicaCount: 3 # This is the amount of how many instances of loolwsd will run
```

Comment:
How do three replicas work properly? 
I thought the application is stateful when multiple users are editing the same file simultaneously?